tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<p>It was recommended to me to not reference the navigation side bar. </p><p> </p><p>You also only need to reference the "Define model" section given my comment that "Train model" is not needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>